robottelo/config/settings.py

```diff
@@ -0,0 +1,365 @@
+"""Define and instantiate the configuration class for Robottelo."""
+import logging
+import os
+import sys
+
+try:
+    from ConfigParser import (
+        NoOptionError,
+        NoSectionError,
+        SafeConfigParser as ConfigParser,
+    )
+except ImportError:
+    from configparser import (
+        ConfigParser,
+        NoOptionError,
+        NoSectionError,
+    )
+
+from logging import config
+from robottelo.config import casts
+# from robottelo.helpers import configure_entities
```

Comment:
This was in place until I get the refactor. Was able to finish the refacor
late today and will finish polishing tomorrow as I have to review the
entire diff.

On Wed, Sep 23, 2015, 23:11 sthirugn <notifications@github.com> wrote:

> In robottelo/config/settings.py
> <https://github.com/SatelliteQE/robottelo/pull/2807#discussion_r40279872>:
>
> > +try:
> > +    from ConfigParser import (
> > +        NoOptionError,
> > +        NoSectionError,
> > +        SafeConfigParser as ConfigParser,
> > +    )
> > +except ImportError:
> > +    from configparser import (
> > +        ConfigParser,
> > +        NoOptionError,
> > +        NoSectionError,
> > +    )
> > +
> > +from logging import config
> > +from robottelo.config import casts
> > +# from robottelo.helpers import configure_entities
>
> remove the comment if not necessary
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SatelliteQE/robottelo/pull/2807/files#r40279872>.
>
-- 
Elyézer Rezende